test/comparator/test_enum_value_comparator.py

```diff
@@ -1,13 +1,17 @@
 import unittest
-import test.testdata.original_pb2 as original_version
-
+from test.tools.invoker import UnittestInvoker
 from src.comparator.enum_value_comparator import EnumValueComparator
 from src.findings.finding_container import FindingContainer
 from src.findings.utils import FindingCategory
 
 class EnumValueComparatorTest(unittest.TestCase):
+    _PROTO_ORIGINAL = 'address_book.proto'
+    _DESCRIPTOR_SET_ORIGINAL = 'address_book_descriptor_set.pb'
+    _INVOKER_ORIGNAL = UnittestInvoker([_PROTO_ORIGINAL], _DESCRIPTOR_SET_ORIGINAL)
+    _PB_ORIGNAL = _INVOKER_ORIGNAL.run()
```

Comment:
Sure, done. This is kind of special. I took two different enum values from one proto file `address_book.proto` instead of two, since we only need two different enumValueDescriptors.